src/index.html

```diff
@@ -2,11 +2,51 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    >
     <title>Calendar</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link
+      rel="stylesheet"
+      href="./styles/main.scss"
+    >
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main class="page">
+      <div class="calendar calendar__month-lenght--30 calendar__start-day--mon">
```

Comment:
```suggestion
      <div class="calendar calendar--30 calendar--mon">
```
Можно вот так оставить

Reply:
Ok. Thank you